src/tests/functional/plugin/gpu/single_layer_tests/eye.cpp

```diff
@@ -0,0 +1,140 @@
+const std::vector<std::vector<int>> emptyBatchShape = {{0}};
+const std::vector<std::vector<int>> batchShapes1D = {{3}, {2}, {1}, {0}};
+const std::vector<std::vector<int>> batchShapes2D = {{3, 2}, {2, 1}, {0, 0}};
+const std::vector<std::vector<int>> batchShapes3D = {{3, 2, 1}, {1, 1, 1}};
+
+INSTANTIATE_TEST_SUITE_P(smoke_Eye2D_WithNonScalar_Test,
+                         EyeLayerGPUTest,
+                         ::testing::Combine(::testing::ValuesIn(std::vector<std::vector<ov::Shape>>{{{1}, {1}, {1}}}),
+                                            ::testing::ValuesIn(emptyBatchShape),
+                                            ::testing::ValuesIn(eyePars),
+                                            ::testing::ValuesIn(netPrecisions),
+                                            ::testing::Values(CommonTestUtils::DEVICE_GPU)),
+                         EyeLayerGPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_Eye_1DBatch_Test,
+                         EyeLayerGPUTest,
+                         ::testing::Combine(::testing::ValuesIn(std::vector<std::vector<ov::Shape>>{
+                                                {{1}, {1}, {1}, {1}}}),
+                                            ::testing::ValuesIn(batchShapes1D),
+                                            ::testing::ValuesIn(eyePars),
+                                            ::testing::ValuesIn(netPrecisions),
+                                            ::testing::Values(CommonTestUtils::DEVICE_GPU)),
+                         EyeLayerGPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_Eye_2DBatch_Test,
+                         EyeLayerGPUTest,
+                         ::testing::Combine(::testing::ValuesIn(std::vector<std::vector<ov::Shape>>{
+                                                {{1}, {1}, {1}, {2}}}),
+                                            ::testing::ValuesIn(batchShapes2D),
+                                            ::testing::ValuesIn(eyePars),
+                                            ::testing::ValuesIn(netPrecisions),
+                                            ::testing::Values(CommonTestUtils::DEVICE_GPU)),
+                         EyeLayerGPUTest::getTestCaseName);
+
+// Ticket: 85127
+INSTANTIATE_TEST_SUITE_P(smoke_Eye_3DBatch_Test,
+                         EyeLayerGPUTest,
+                         ::testing::Combine(::testing::ValuesIn(std::vector<std::vector<ov::Shape>>{
+                                                {{1}, {1}, {1}, {3}}}),
+                                            ::testing::ValuesIn(batchShapes3D),
+                                            ::testing::ValuesIn(eyePars),
+                                            ::testing::ValuesIn(netPrecisions),
+                                            ::testing::Values(CommonTestUtils::DEVICE_GPU)),
+                         EyeLayerGPUTest::getTestCaseName);
+
+// TODO Dynamic shapes are not suppoted by GPU pluging
```

Comment:
I beleive we can remove  it for now and add it together with dynamic shapes support (as you got this data from CPU plugin, right?)